src/win/tty.c

```diff
@@ -1871,133 +1911,160 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
               continue;
             }
 
-          } else if (utf8_codepoint == '?' && !(ansi_parser_state & ANSI_IN_ARG) &&
```

Comment:
Why did you eliminate the `!(ansi_parser_state & ANSI_IN_ARG) &&  handle->tty.wr.ansi_csi_argc == 0` condition here?